test/Microsoft.TestPlatform.ObjectModel.PlatformTests/DiaSessionTests.cs

```diff
@@ -8,17 +8,17 @@
 using Microsoft.VisualStudio.TestPlatform.ObjectModel;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-#nullable disable
-
 namespace Microsoft.TestPlatform.ObjectModel.PlatformTests;
 
 [TestClass]
 public class DiaSessionTests : IntegrationTestBase
 {
+#if NETFRAMEWORK
     private const string NET451 = "net451";
+#endif
     private const string NETCOREAPP21 = "netcoreapp2.1";
```

Comment:
Why is net451 in compiler directive but netcoreapp21 is not?

Reply:
I probably got the warning only for one part. I will update.